src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const input = sourceString.split(';');
+  const result = {};
+
+  input.forEach((item) => {
+    const [key, value] = item.split(':');
+
+    if (value !== undefined) {
```

Comment:
```suggestion
    if (value) {
```
You don't need to compare with undefined, just check if a value exists